specification/compute/resource-manager/Microsoft.Compute/stable/2020-12-01/compute.json

```diff
@@ -3247,10 +3250,11 @@
             "type": "string",
             "description": "The expand expression to apply on the operation.",
             "enum": [
-              "instanceView"
+              "instanceView",
+              "userData"
             ],
             "x-ms-enum": {
-              "name": "InstanceViewTypes",
```

Comment:
@roipmsft , changing the name here is a breaking change. Please revert this change and edit the backend if needed.

Reply:
I am still fixing the PR please dont review right now

Reply:
reverted as you suggestedty